src/cmd.zig

```diff
@@ -1214,7 +1226,31 @@ fn validator(
     );
     defer shred_network_manager.deinit();
 
-    const maybe_vote_sockets: ?replay.consensus.core.VoteSockets = if (cfg.voting_enabled)
+    // Vote account. if not provided, disable voting
+    const maybe_vote_pubkey: ?Pubkey = if (cfg.voting_enabled) blk: {
+        const vote_keypair_path = cfg.vote_account_path orelse default_path: {
+            const app_data_dir_path = try std.fs.getAppDataDir(allocator, "sig");
+            defer allocator.free(app_data_dir_path);
+            break :default_path try std.fs.path.join(
+                allocator,
+                &.{ app_data_dir_path, "vote-account.key" },
+            );
+        };
+        defer if (cfg.vote_account_path == null) allocator.free(vote_keypair_path);
+
+        break :blk sig.identity.readBinaryKeypairPubkey(vote_keypair_path) catch |err| {
+            app_base.logger.warn().logf(
```

Comment:
i think error makes more sense instead of warn here. actually maybe the validator should exit if the vote key is invalid. the user explicitly specified an invalid input, and sig can't follow the instructions. why not just use `try`?